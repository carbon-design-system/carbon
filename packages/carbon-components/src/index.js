/**
 * Copyright IBM Corp. 2016, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

// ====================//
// Imports and Exports //
// ====================//

// Base Elements & Components
// -------------
// - JavaScript classes for use with components and base-elements.
// - The following statements import classes from actual locations to
//   be consumed from this file instead of their actual locations.
export * from './globals/js/components';

export { default as settings } from './globals/js/settings';
