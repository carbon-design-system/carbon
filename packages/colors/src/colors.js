/**
 * Copyright IBM Corp. 2018, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

export const black = '#000000';
export const black100 = black;

export const white = '#ffffff';
export const white0 = white;

export const yellow = '#f1c21b';
export const yellow30 = yellow;
export const yellow40 = '#d2a106';
export const yellow50 = '#b28600';

export const orange = '#ff832b';
export const orange40 = orange;
export const orange60 = '#ba4e00';
export const orange70 = '#8a3800';

export const red10 = '#fff1f1';
export const red20 = '#ffd7d9';
export const red30 = '#ffb3b8';
export const red40 = '#ff8389';
export const red50 = '#fa4d56';
export const red60 = '#da1e28';
export const red70 = '#a2191f';
export const red80 = '#750e13';
export const red90 = '#520408';
export const red100 = '#2d0709';
export const red = {
  10: red10,
  20: red20,
  30: red30,
  40: red40,
  50: red50,
  60: red60,
  70: red70,
  80: red80,
  90: red90,
  100: red100,
};

export const magenta10 = '#fff0f7';
export const magenta20 = '#ffd6e8';
export const magenta30 = '#ffafd2';
export const magenta40 = '#ff7eb6';
export const magenta50 = '#ee5396';
export const magenta60 = '#d02670';
export const magenta70 = '#9f1853';
export const magenta80 = '#740937';
export const magenta90 = '#510224';
export const magenta100 = '#2a0a18';
export const magenta = {
  10: magenta10,
  20: magenta20,
  30: magenta30,
  40: magenta40,
  50: magenta50,
  60: magenta60,
  70: magenta70,
  80: magenta80,
  90: magenta90,
  100: magenta100,
};

export const purple10 = '#f6f2ff';
export const purple20 = '#e8daff';
export const purple30 = '#d4bbff';
export const purple40 = '#be95ff';
export const purple50 = '#a56eff';
export const purple60 = '#8a3ffc';
export const purple70 = '#6929c4';
export const purple80 = '#491d8b';
export const purple90 = '#31135e';
export const purple100 = '#1c0f30';
export const purple = {
  10: purple10,
  20: purple20,
  30: purple30,
  40: purple40,
  50: purple50,
  60: purple60,
  70: purple70,
  80: purple80,
  90: purple90,
  100: purple100,
};

export const blue10 = '#edf5ff';
export const blue20 = '#d0e2ff';
export const blue30 = '#a6c8ff';
export const blue40 = '#78a9ff';
export const blue50 = '#4589ff';
export const blue60 = '#0f62fe';
export const blue70 = '#0043ce';
export const blue80 = '#002d9c';
export const blue90 = '#001d6c';
export const blue100 = '#001141';
export const blue = {
  10: blue10,
  20: blue20,
  30: blue30,
  40: blue40,
  50: blue50,
  60: blue60,
  70: blue70,
  80: blue80,
  90: blue90,
  100: blue100,
};

export const cyan10 = '#e5f6ff';
export const cyan20 = '#bae6ff';
export const cyan30 = '#82cfff';
export const cyan40 = '#33b1ff';
export const cyan50 = '#1192e8';
export const cyan60 = '#0072c3';
export const cyan70 = '#00539a';
export const cyan80 = '#003a6d';
export const cyan90 = '#012749';
export const cyan100 = '#061727';
export const cyan = {
  10: cyan10,
  20: cyan20,
  30: cyan30,
  40: cyan40,
  50: cyan50,
  60: cyan60,
  70: cyan70,
  80: cyan80,
  90: cyan90,
  100: cyan100,
};

export const teal10 = '#d9fbfb';
export const teal20 = '#9ef0f0';
export const teal30 = '#3ddbd9';
export const teal40 = '#08bdba';
export const teal50 = '#009d9a';
export const teal60 = '#007d79';
export const teal70 = '#005d5d';
export const teal80 = '#004144';
export const teal90 = '#022b30';
export const teal100 = '#081a1c';
export const teal = {
  10: teal10,
  20: teal20,
  30: teal30,
  40: teal40,
  50: teal50,
  60: teal60,
  70: teal70,
  80: teal80,
  90: teal90,
  100: teal100,
};

export const green10 = '#defbe6';
export const green20 = '#a7f0ba';
export const green30 = '#6fdc8c';
export const green40 = '#42be65';
export const green50 = '#24a148';
export const green60 = '#198038';
export const green70 = '#0e6027';
export const green80 = '#044317';
export const green90 = '#022d0d';
export const green100 = '#071908';
export const green = {
  10: green10,
  20: green20,
  30: green30,
  40: green40,
  50: green50,
  60: green60,
  70: green70,
  80: green80,
  90: green90,
  100: green100,
};

export const coolGray10 = '#f2f4f8';
export const coolGray20 = '#dde1e6';
export const coolGray30 = '#c1c7cd';
export const coolGray40 = '#a2a9b0';
export const coolGray50 = '#878d96';
export const coolGray60 = '#697077';
export const coolGray70 = '#4d5358';
export const coolGray80 = '#343a3f';
export const coolGray90 = '#21272a';
export const coolGray100 = '#121619';
export const coolGray = {
  10: coolGray10,
  20: coolGray20,
  30: coolGray30,
  40: coolGray40,
  50: coolGray50,
  60: coolGray60,
  70: coolGray70,
  80: coolGray80,
  90: coolGray90,
  100: coolGray100,
};

export const gray10 = '#f4f4f4';
export const gray20 = '#e0e0e0';
export const gray30 = '#c6c6c6';
export const gray40 = '#a8a8a8';
export const gray50 = '#8d8d8d';
export const gray60 = '#6f6f6f';
export const gray70 = '#525252';
export const gray80 = '#393939';
export const gray90 = '#262626';
export const gray100 = '#161616';
export const gray = {
  10: gray10,
  20: gray20,
  30: gray30,
  40: gray40,
  50: gray50,
  60: gray60,
  70: gray70,
  80: gray80,
  90: gray90,
  100: gray100,
};

export const warmGray10 = '#f7f3f2';
export const warmGray20 = '#e5e0df';
export const warmGray30 = '#cac5c4';
export const warmGray40 = '#ada8a8';
export const warmGray50 = '#8f8b8b';
export const warmGray60 = '#726e6e';
export const warmGray70 = '#565151';
export const warmGray80 = '#3c3838';
export const warmGray90 = '#272525';
export const warmGray100 = '#171414';
export const warmGray = {
  10: warmGray10,
  20: warmGray20,
  30: warmGray30,
  40: warmGray40,
  50: warmGray50,
  60: warmGray60,
  70: warmGray70,
  80: warmGray80,
  90: warmGray90,
  100: warmGray100,
};

// Deprecated ☠️
export const yellow20 = '#fdd13a';

export const colors = {
  black: {
    100: black,
  },
  blue,
  coolGray,
  cyan,
  gray,
  green,
  magenta,
  orange: {
    40: orange40,
    60: orange60,
    70: orange70,
  },
  purple,
  red,
  teal,
  warmGray,
  white: {
    0: white,
  },
  yellow: {
    20: yellow20,
    30: yellow30,
    40: yellow40,
    50: yellow50,
  },
};
