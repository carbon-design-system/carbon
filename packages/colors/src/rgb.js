/**
 * Copyright IBM Corp. 2018, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

/**
 * Parse a given hexcode string into an rgba statement with the given opacity
 * @param {string} hexcode
 * @param {number} opacity
 * @returns {string}
 */
export function rgba(hexcode, opacity) {
  const values = [
    hexcode.substring(1, 3),
    hexcode.substring(3, 5),
    hexcode.substring(5, 7),
  ].map((string) => parseInt(string, 16));
  return `rgba(${values[0]}, ${values[1]}, ${values[2]}, ${opacity})`;
}
