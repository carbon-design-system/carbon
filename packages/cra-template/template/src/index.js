import React from 'react';
import ReactDOM from 'react-dom';
import './index.scss';
import App from './App';
import reportWebVitals from './reportWebVitals';
import { ApolloProvider, ApolloClient, InMemoryCache } from '@apollo/client';
const cache = new InMemoryCache();

const client = new ApolloClient({
  cache: cache,
  uri: 'https://api.github.com/graphql',
  headers: {
    authorization: `Bearer ${process.env.REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN}`,
  },
});

ReactDOM.render(
  <React.StrictMode>
    <ApolloProvider client={client}>
      <App />
    </ApolloProvider>
  </React.StrictMode>,
  document.getElementById('root')
);

// If you want to start measuring performance in your app, pass a function
// to log results (for example: reportWebVitals(console.log))
// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
reportWebVitals();
