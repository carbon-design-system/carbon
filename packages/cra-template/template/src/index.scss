@use '@carbon/react';
@use '@carbon/react/scss/breakpoint' as *;
