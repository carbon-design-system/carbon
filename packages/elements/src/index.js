/**
 * Copyright IBM Corp. 2018, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

export {
  black,
  black100,
  blue,
  blue10,
  blue20,
  blue30,
  blue40,
  blue50,
  blue60,
  blue70,
  blue80,
  blue90,
  blue100,
  colors,
  coolGray,
  coolGray10,
  coolGray20,
  coolGray30,
  coolGray40,
  coolGray50,
  coolGray60,
  coolGray70,
  coolGray80,
  coolGray90,
  coolGray100,
  cyan,
  cyan10,
  cyan20,
  cyan30,
  cyan40,
  cyan50,
  cyan60,
  cyan70,
  cyan80,
  cyan90,
  cyan100,
  gray,
  gray10,
  gray20,
  gray30,
  gray40,
  gray50,
  gray60,
  gray70,
  gray80,
  gray90,
  gray100,
  green,
  green10,
  green20,
  green30,
  green40,
  green50,
  green60,
  green70,
  green80,
  green90,
  green100,
  magenta,
  magenta10,
  magenta20,
  magenta30,
  magenta40,
  magenta50,
  magenta60,
  magenta70,
  magenta80,
  magenta90,
  magenta100,
  orange,
  orange40,
  orange60,
  orange70,
  purple,
  purple10,
  purple20,
  purple30,
  purple40,
  purple50,
  purple60,
  purple70,
  purple80,
  purple90,
  purple100,
  red,
  red10,
  red20,
  red30,
  red40,
  red50,
  red60,
  red70,
  red80,
  red90,
  red100,
  rgba,
  teal,
  teal10,
  teal20,
  teal30,
  teal40,
  teal50,
  teal60,
  teal70,
  teal80,
  teal90,
  teal100,
  warmGray,
  warmGray10,
  warmGray20,
  warmGray30,
  warmGray40,
  warmGray50,
  warmGray60,
  warmGray70,
  warmGray80,
  warmGray90,
  warmGray100,
  white0,
  yellow,
  yellow20,
  yellow30,
  yellow40,
  yellow50,
} from '@carbon/colors';
export {
  fluid,
  fontFamilies,
  fontFamily,
  fontWeights,
  fontWeight,
  print,
  reset,
  getTypeSize,
  scale,
  styles,
  unstable_tokens,
} from '@carbon/type';
export {
  baseFontSize,
  rem,
  em,
  px,
  breakpoints,
  breakpointUp,
  breakpointDown,
  breakpoint,
  container,
  fluidSpacing,
  iconSize,
  miniUnit,
  miniUnits,
  layout,
  spacing,
} from '@carbon/layout';
export * from '@carbon/themes';
export {
  fast01,
  fast02,
  moderate01,
  moderate02,
  slow01,
  slow02,
  easings,
  motion,
} from '@carbon/motion';
