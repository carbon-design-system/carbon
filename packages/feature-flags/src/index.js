/**
 * Copyright IBM Corp. 2015, 2020
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import { featureFlagInfo } from './generated/feature-flags';

const featureFlags = new Map();

for (let i = 0; i < featureFlagInfo.length; i++) {
  const featureFlag = featureFlagInfo[i];
  featureFlags.set(featureFlag.name, featureFlag.enabled);
}

/**
 * Check to see if a flag exists
 * @param {string} name
 */
function checkForFlag(name) {
  if (!featureFlags.has(name)) {
    throw new Error(`Unable to find a feature flag with the name \`${name}\``);
  }
}

/**
 * Add a feature flag
 * @param {string} name
 * @param {boolean} enabled
 */
export function add(name, enabled) {
  if (featureFlags.has(name)) {
    throw new Error(`The feature flag: ${name} already exists`);
  }
  featureFlags.set(name, enabled);
}

/**
 * Enable a feature flag
 * @param {string} name
 */
export function enable(name) {
  checkForFlag(name);
  featureFlags.set(name, true);
}

/**
 * Disable a feature flag
 * @param {string} name
 */
export function disable(name) {
  checkForFlag(name);
  featureFlags.set(name, false);
}

/**
 * Merge the given feature flags with the current set of feature flags.
 * Duplicate keys will be set to the value in the given feature flags.
 * @param {object} flags
 */
export function merge(flags) {
  Object.keys(flags).forEach((key) => {
    featureFlags.set(key, flags[key]);
  });
}

/**
 * Check if a feature flag is enabled
 * @param {string} name
 * @returns {boolean}
 */
export function enabled(name) {
  checkForFlag(name);
  return featureFlags.get(name);
}

export { featureFlagInfo as unstable_featureFlagInfo };
