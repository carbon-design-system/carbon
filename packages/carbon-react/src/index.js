/**
 * Copyright IBM Corp. 2016, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import './feature-flags';

export {
  Accordion,
  AccordionItem,
  AspectRatio,
  Breadcrumb,
  BreadcrumbItem,
  Button,
  ButtonSet,
  Checkbox,
  CodeSnippet,
  ComboBox,
  ComposedModal,
  ModalHeader,
  ModalBody,
  ModalFooter,
  ContentSwitcher,
  Copy,
  CopyButton,
  DangerButton,
  DataTable,
  Table,
  TableActionList,
  TableBatchAction,
  TableBatchActions,
  TableBody,
  TableCell,
  TableContainer,
  TableExpandHeader,
  TableExpandRow,
  TableExpandedRow,
  TableHead,
  TableHeader,
  TableRow,
  TableSelectAll,
  TableSelectRow,
  TableToolbar,
  TableToolbarAction,
  TableToolbarContent,
  TableToolbarSearch,
  TableToolbarMenu,
  DatePicker,
  DatePickerInput,
  Dropdown,
  ErrorBoundary,
  ErrorBoundaryContext,
  FileUploader,
  Filename,
  FileUploaderButton,
  FileUploaderDropContainer,
  FileUploaderItem,
  Form,
  FluidForm,
  FormGroup,
  FormItem,
  FormLabel,
  Row,
  InlineLoading,
  Link,
  ListItem,
  Loading,
  Modal,
  MultiSelect,
  ToastNotification,
  InlineNotification,
  NotificationActionButton,
  NotificationButton,
  NotificationTextDetails,
  NumberInput,
  OrderedList,
  OverflowMenu,
  OverflowMenuItem,
  Pagination,
  PaginationNav,
  PrimaryButton,
  ProgressIndicator,
  ProgressStep,
  RadioButton,
  RadioButtonGroup,
  Search,
  SearchFilterButton,
  SearchLayoutButton,
  SecondaryButton,
  Select,
  SelectItem,
  SelectItemGroup,
  Switch,
  Slider,
  StructuredListWrapper,
  StructuredListHead,
  StructuredListBody,
  StructuredListRow,
  StructuredListInput,
  StructuredListCell,
  Tab,
  TabContent,
  Tabs,
  Tag,
  TextArea,
  TextInput,
  Tile,
  ClickableTile,
  SelectableTile,
  ExpandableTile,
  TileAboveTheFoldContent,
  TileBelowTheFoldContent,
  RadioTile,
  TileGroup,
  TimePicker,
  TimePickerSelect,
  Toggle,
  ToggleSmall,
  Toolbar,
  ToolbarItem,
  ToolbarTitle,
  ToolbarOption,
  ToolbarDivider,
  ToolbarSearch,
  Tooltip,
  TooltipDefinition,
  TooltipIcon,
  UnorderedList,
  SkeletonText,
  SkeletonPlaceholder,
  DataTableSkeleton,
  AccordionSkeleton,
  BreadcrumbSkeleton,
  ButtonSkeleton,
  CheckboxSkeleton,
  CodeSnippetSkeleton,
  DropdownSkeleton,
  FileUploaderSkeleton,
  NumberInputSkeleton,
  PaginationSkeleton,
  ProgressIndicatorSkeleton,
  RadioButtonSkeleton,
  SearchSkeleton,
  SelectSkeleton,
  SliderSkeleton,
  StructuredListSkeleton,
  TabsSkeleton,
  TagSkeleton,
  TextAreaSkeleton,
  TextInputSkeleton,
  ToggleSkeleton,
  ToggleSmallSkeleton,
  IconSkeleton,
  DatePickerSkeleton,
  Content,
  Header,
  HeaderContainer,
  HeaderGlobalAction,
  HeaderGlobalBar,
  HeaderMenu,
  HeaderMenuButton,
  HeaderMenuItem,
  HeaderName,
  HeaderNavigation,
  HeaderPanel,
  HeaderSideNavItems,
  Switcher,
  SwitcherItem,
  SwitcherDivider,
  SkipToContent,
  SideNav,
  SideNavDetails,
  SideNavDivider,
  SideNavFooter,
  SideNavHeader,
  SideNavIcon,
  SideNavItem,
  SideNavItems,
  SideNavLink,
  SideNavLinkText,
  SideNavMenu,
  SideNavMenuItem,
  SideNavSwitcher,
  // Experimental
  unstable_PageSelector,
  unstable_Pagination,
  unstable_TreeView,
  unstable_TreeNode,
  unstable_ContextMenu,
  unstable_ContextMenuDivider,
  unstable_ContextMenuGroup,
  unstable_ContextMenuItem,
  unstable_ContextMenuRadioGroup,
  unstable_ContextMenuSelectableItem,
  unstable_useContextMenu,
  unstable_Heading,
  unstable_Section,
} from 'carbon-components-react';

export {
  // Grid
  Grid,
  Column,
} from './components/Grid';
