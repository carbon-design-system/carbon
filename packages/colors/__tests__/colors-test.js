/**
 * Copyright IBM Corp. 2015, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @jest-environment node
 */

'use strict';

import { colors, hoverColors } from '../src';

test('colors', () => {
  expect(colors).toMatchSnapshot();
});

test('hoverColors', () => {
  expect(hoverColors).toMatchSnapshot();
});
