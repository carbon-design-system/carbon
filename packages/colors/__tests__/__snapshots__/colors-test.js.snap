// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors 1`] = `
Object {
  "black": Object {
    "100": "#000000",
  },
  "blue": Object {
    "10": "#edf5ff",
    "100": "#001141",
    "20": "#d0e2ff",
    "30": "#a6c8ff",
    "40": "#78a9ff",
    "50": "#4589ff",
    "60": "#0f62fe",
    "70": "#0043ce",
    "80": "#002d9c",
    "90": "#001d6c",
  },
  "coolGray": Object {
    "10": "#f2f4f8",
    "100": "#121619",
    "20": "#dde1e6",
    "30": "#c1c7cd",
    "40": "#a2a9b0",
    "50": "#878d96",
    "60": "#697077",
    "70": "#4d5358",
    "80": "#343a3f",
    "90": "#21272a",
  },
  "cyan": Object {
    "10": "#e5f6ff",
    "100": "#061727",
    "20": "#bae6ff",
    "30": "#82cfff",
    "40": "#33b1ff",
    "50": "#1192e8",
    "60": "#0072c3",
    "70": "#00539a",
    "80": "#003a6d",
    "90": "#012749",
  },
  "gray": Object {
    "10": "#f4f4f4",
    "100": "#161616",
    "20": "#e0e0e0",
    "30": "#c6c6c6",
    "40": "#a8a8a8",
    "50": "#8d8d8d",
    "60": "#6f6f6f",
    "70": "#525252",
    "80": "#393939",
    "90": "#262626",
  },
  "green": Object {
    "10": "#defbe6",
    "100": "#071908",
    "20": "#a7f0ba",
    "30": "#6fdc8c",
    "40": "#42be65",
    "50": "#24a148",
    "60": "#198038",
    "70": "#0e6027",
    "80": "#044317",
    "90": "#022d0d",
  },
  "magenta": Object {
    "10": "#fff0f7",
    "100": "#2a0a18",
    "20": "#ffd6e8",
    "30": "#ffafd2",
    "40": "#ff7eb6",
    "50": "#ee5396",
    "60": "#d02670",
    "70": "#9f1853",
    "80": "#740937",
    "90": "#510224",
  },
  "orange": Object {
    "40": "#ff832b",
    "60": "#ba4e00",
    "70": "#8a3800",
  },
  "purple": Object {
    "10": "#f6f2ff",
    "100": "#1c0f30",
    "20": "#e8daff",
    "30": "#d4bbff",
    "40": "#be95ff",
    "50": "#a56eff",
    "60": "#8a3ffc",
    "70": "#6929c4",
    "80": "#491d8b",
    "90": "#31135e",
  },
  "red": Object {
    "10": "#fff1f1",
    "100": "#2d0709",
    "20": "#ffd7d9",
    "30": "#ffb3b8",
    "40": "#ff8389",
    "50": "#fa4d56",
    "60": "#da1e28",
    "70": "#a2191f",
    "80": "#750e13",
    "90": "#520408",
  },
  "teal": Object {
    "10": "#d9fbfb",
    "100": "#081a1c",
    "20": "#9ef0f0",
    "30": "#3ddbd9",
    "40": "#08bdba",
    "50": "#009d9a",
    "60": "#007d79",
    "70": "#005d5d",
    "80": "#004144",
    "90": "#022b30",
  },
  "warmGray": Object {
    "10": "#f7f3f2",
    "100": "#171414",
    "20": "#e5e0df",
    "30": "#cac5c4",
    "40": "#ada8a8",
    "50": "#8f8b8b",
    "60": "#726e6e",
    "70": "#565151",
    "80": "#3c3838",
    "90": "#272525",
  },
  "white": Object {
    "0": "#ffffff",
  },
  "yellow": Object {
    "20": "#fdd13a",
    "30": "#f1c21b",
    "40": "#d2a106",
    "50": "#b28600",
  },
}
`;
